light_curves/code_src/sample_selection.py

```diff
@@ -316,6 +318,30 @@ def get_graham_sample(coords, labels, *, verbose=1):
         print('Changing Look AGN- Graham et al: ',len(CSQ))
 
 
+def get_tde_sample(coords, labels, *, object_ids=["ZTF18aabtxvd", "ZTF18aahqkbt"], verbose=1):
+    """ To find and append coordinates of objects with only ZTF obj name
+
+    Parameters
+    ----------
+    coords : list of astropy skycoords
+        the coordinates of the targets for which a user wants light curves
+    labels: list of strings
+        journal articles associated with the target coordinates
+    object_ids: list of strings
+        eg., [ "ZTF18accqogs", "ZTF19aakyhxi", "ZTF19abyylzv", "ZTF19acyfpno"]
+    verbose: int
+        print out debugging info (1) or not(0)
+    """
+    alerce = Alerce()
+    objects = alerce.query_objects(oid=object_ids, format="pandas")
+    tde_coords = [SkyCoord(ra, dec, frame='icrs', unit='deg') for ra, dec in zip(objects['meanra'], objects['meandec'])]
+    tde_labels = ['ZTF-Objname' for _ in objects['meanra']]
```

Comment:
@xoubish these presumably TDEs are coming in with label = 'ZTF..'.  Should they have a label which includes the word 'TDE', or should we rename the function to be something broader like 'get_ZTF_objname', since the function looks like it doesn't know anything about what sort of objects these are.

Reply:
Yes, a general name would be better.